frontend/src/metabase/lib/api.js

```diff
@@ -6,11 +6,19 @@ import EventEmitter from "events";
 
```

Comment:
FYI the changes to this file aren't relevant to this PR (it was to an earlier iteration), but it's something I want to add anyway.